test/matchtests.jl

```diff
@@ -209,36 +209,71 @@ end
 
 @assert @match([1 2 3 4; 5 6 7 8], [a b... c])              == ([1,5] , [2 3; 6 7] , [4,8])
 
-# match / collect rows
-@assert @match([1 2 3; 4 5 6], [a, b])                      == ([1 2 3], [4 5 6])
-@assert @match([1 2 3; 4 5 6], [[1 2 3], a])                ==  [4 5 6]             # TODO: don't match this
-@assert @match([1 2 3; 4 5 6], [1 2 3; a])                  ==  [4 5 6]
+if VERSION < v"0.5.0-dev"
```

Comment:
Thanks @tkelman.  I probably should have just commented these out, but I added this code long before I decided that this release would only support v0.5.  The last version that supports v0.4 still includes them (without qualification on `VERSION`), and this section won't run on v0.5* or v0.6.

These tests broke because of changes in the representation of arrays in Julia.  I could try to figure out when that actually happened, but I'd rather just try to make them work again, and I'll only go back and bisect if I can't figure out how to fix them.